lighthouse-core/test/chromium-web-tests/download-devtools.sh

```diff
@@ -24,12 +24,11 @@ then
     git pull --ff-only -f origin main
```

Comment:
i know these two branches are the same but I don't care. do you really want to read a compound bash logic if? because I don't want to write one.